src/htmltojsx.js

```diff
@@ -497,6 +514,10 @@ StyleParser.prototype = {
    * @return {string} JSX style key
    */
   toJSXKey: function(key) {
+    // Don't capitalize -ms- prefix
+    if(key.startsWith('-ms-')) {
```

Comment:
[`startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) is not yet standard and not widely supported.